.gitignore

```diff
@@ -7,6 +7,7 @@ build/jacocoHtml/
 build/reports/tests/
 build/scripts/
 build/tmp/
+logs.txt
```

Comment:
I see that you have also accidentally committed `testLogs.txt` and `testLoggerLogs.txt` below. Can you think of a way to ignore all log files rather than specific log files?

Reply:
moved to gitignore in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/eeef1cc27cafdd69873c1ff48e4fe135e672db2c